docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,292 @@
+
+NOTE: Blue-Green deployment is currently supported by Hazelcast's Java, Node.js, and Go clients.
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Trial license
+|https://trialrequest.hazelcast.com/[Hazelcast website]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|Docker network with the name `hazelcast-network`
+|Use the `docker network create hazelcast-network` command 
+
+|===
+
+== Step 1. Start the Blue Enterprise Cluster
+
+In this step, you use the Hazelcast Enterprise Docker image to start a local single-member cluster called `blue`.
+This step also installs your Enterprise license key.
+
+Run the following command on the terminal:
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --network hazelcast-network \
+    --rm \
+    -e HZ_CLUSTERNAME=blue \
+    -e HZ_LICENSEKEY=<your license key> \ <1>
+    -p 5701:5701 hazelcast/hazelcast-enterprise:{full-version}
+----
+<1> Replace the `<your license key>` placeholder with your Hazelcast Enterprise license key.
+
+You should see your cluster name in the console along with the IP address of the Docker container that's running the Hazelcast member.
+
+[source,shell,subs="attributes+"]
+----
+2021-12-01 18:26:42,369 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.2]:5701 [blue] [{full-version}] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.2]:5701 - c00213e1-da50-4b5f-a53b-ccfe4a1ebeea this
+]
+
+2021-12-01 18:26:42,384 [ INFO] [main] [c.h.c.LifecycleService]: [172.18.0.2]:5701 [blue] [{full-version}] [172.18.0.2]:5701 is STARTED
+----
+
+== Step 2. Start the Green Enterprise Cluster
+
+Start another local single-member cluster called `green`.
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --network hazelcast-network \
+    --rm \
+    -e HZ_CLUSTERNAME=green \
+    -e HZ_LICENSEKEY=<your license key> \ <1>
+    -p 5701:5701 hazelcast/hazelcast-enterprise:{full-version}
+----
+<1> Replace the `<your license key>` placeholder with your Hazelcast Enterprise license key.
+
+See the `green` cluster is formed:
+
+[source,shell]
+----
+2021-12-01 18:28:46,299 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.3]:5701 [green] [5.0] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.3]:5701 - 72f5520c-8c27-4501-9199-a8da6b58c0b4 this
+]
+
+2021-12-01 18:28:46,321 [ INFO] [main] [c.h.c.LifecycleService]: [172.18.0.3]:5701 [green] [5.0] [172.18.0.3]:5701 is STARTED
+----
+
+Now, you have two separate Hazelcast Enterprise clusters running on your local.
+
+== Step 3. Configure the Client
+
+In this step, you'll configure a client, so that it initially connects to the `blue` cluster, and when
+the `blue` cluster is down, it automatically connects to the `green` cluster.
+
+For this, you need to create two client configurations for the same client, and pass these to a failover configuration.
```

Comment:
Creating the config files doesn't work for the Node.js client. We should only use these steps in the Java tab. It would also be useful to provide an example of what the working directory should look like. Where, in relation to the `.java` file should the configuration files be saved?